docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -509,7 +510,7 @@ Usage Notes
   
   ::
   
-    $cron=$lib.cron.get(<cron_iden>) $cron.set(user, <new_user_iden>)
+    $cron=$lib.cron.get(<cron_iden>) $cron.set(user, <new_creator_iden>)
```

Comment:
the param name on a cron is actually 'creator' instead of 'user' (different than triggers which is probably confusing and maybe we should fix)

Reply:
WORDS ARE HARD